udp.md

```diff
@@ -0,0 +1,10 @@
+## Setup
+
+For more details about the UDP input settings, check the [Filebeat documentation](https://www.elastic.co/docs/reference/beats/filebeat/filebeat-input-udp).
+
+### Collecting logs from UDP
+
+To collect logs via UDP, select **Collect logs via UDP** and configure the following parameters:
+
+- Host
+- Port
```

Comment:
Same comment as `tcp`. There are fewer options for UDP, but all of them are more advanced and the user should refer to the input docs for more info.